source/stylesheets/refills/_grid.scss

```diff
@@ -0,0 +1,23 @@
+.grid {
+  $grid-gutter: 0.1rem;
+
+  display: flex;
+  flex-flow: row wrap;
+  margin: 0 - $grid-gutter;
+
+  .grid-box-wrap {
+    flex-basis: 25%;
+    flex-grow: 1;
+    flex-shrink: 0;
+    min-width: 200px;
+    overflow: hidden;
+    padding: 0 $grid-gutter ($grid-gutter * 2);
+  }
+
+  .grid-box {
+    background: whitesmoke;
```

Comment:
@tysongach also here I know... However I feel like using some very "easy-to-recognize" names like this would make it easier for the user to see where variables should be changed. My idea is to use `whitesmoke` as a default light gray in all Empites. Easy to find, easy to replace. Thoughts on this?